frontend/lib/data-driven-onboarding/data-driven-onboarding.tsx

```diff
@@ -513,9 +513,9 @@ const ACTION_CARDS: ActionCardPropsCreator[] = [
     // Default content temporarily implemented during COVID-19 Outbreak
     const covidMessage = (
       <Trans id="justfix.ddoEfnycCovidMessage">
```

Comment:
Ooo can you change `justfix.ddoEfnycCovidMessage` to be `justfix.ddoEfnycCovidMessage_v2`?  Otherwise I believe CrowdIn will just see that the ID hasn't changed, and won't prompt Johanna to translate this new text to Spanish.

Reply:
Oh wait... Uhh I'm noticing now that you just edited the Spanish translation in the `.po` file already, which I assume you'll be uploading to Crowdin so it's updated on its end.  If that's the case then don't worry about renaming the `id`!